src/components/Footer/styles.scss

```diff
@@ -5,7 +5,7 @@
   height: 100%;
   background-color: colors.$black;
   color: colors.$white;
-  z-index: -1;
+  z-index: 1;
```

Comment:
Does this work if we change the z-index to 0 instead? I think the default index for all elements is 0 so setting it to -1 would make it unclickable because it's underneath technically but the same index should work. That was we don't have to set the calendar to 2 and the order is still correct without introducing too many new layers.